x/fungible/keeper/keeper.go

```diff
@@ -13,7 +13,7 @@ import (
 
 type (
 	Keeper struct {
-		cdc             codec.BinaryCodec
+		cdc             codec.Codec
```

Comment:
want to know: switching `BinaryCodec` to `Codec` won't affect mainnet data decoding?

Reply:
I don't think it should impact 
https://github.com/zeta-chain/node/pull/3095#discussion_r1841464621